www/docs/customization/announce.md

```diff
@@ -213,3 +213,24 @@ announce:
     # Defaults to `{{ .ProjectName }} {{ .Tag }} is out!`
     subject_template: ''GoReleaser {{ .Tag }} was just released!''
 ```
+
+## LinkedIn
```

Comment:
maybe explain it will only post into your own wall, and that we don't support posting in groups et al?